src/reactviews/pages/QueryResult/table/plugins/cellSelectionModel.plugin.ts

```diff
@@ -132,6 +149,14 @@ export class CellSelectionModel<T extends Slick.SlickData>
 
         this.ranges = this.removeInvalidRanges(ranges);
         this.onSelectedRangesChanged.notify(this.ranges);
+        this.webViewState.state.selection = JSON.parse(
```

Comment:
do you think this could introduce perf delays with large resultsets?  Could you please test with very large resultsets (like cross join "sys.objects" a few times of something, and then select all)?  If it's too slow, maybe there's a way to avoid the deep copy here?

Reply:
If we need this selection info to be included in the state, we might just include it into the `QueryResultWebviewState` definition (or did I miss this?).
And ADS keeps selection for each grid, do we want this or just keep selection for the grid in focus?
<img width="913" alt="image" src="https://github.com/user-attachments/assets/35e25d8a-20f3-4104-888e-bf78c194723e">

Reply:
I did add it to the QueryResultWebviewState definition.  I think we should keep a selection for each grid, since we'll eventually want to be able to copy results from multiple grids at once.